test/cloud_testing/platforms/centos7_x86_64_setup.sh

```diff
@@ -55,6 +55,9 @@ install_from_repo tree
 # traffic shaping
 install_from_repo trickle
 
+# Install test dependency for 647
+install_from_repo python-flask          || die "fail (installing python-flask)"
```

Comment:
I think this test is enabled by default on all platforms (all tests `src/6*` are run).
We have two options: either the python-flask should be added to the other platform setup scripts, or the tests should be blacklisted on other platforms, if it's only relevant to SLC6 and CC7

Reply:
This test is similar to the x509 tests.  I believe that test is only enabled on slc6.  I would like the 647 test to run only on slc6 and CC7.

Where do I specify the blacklist?

Reply:
Each integration platform has a setup and a test script. In the test scripts the `run.sh` script runs the various test suites. For example, in the CC7 test script, the server integration scripts are started as follows:
```
./run.sh $SERVER_TEST_LOGFILE -o ${SERVER_TEST_LOGFILE}${XUNIT_OUTPUT_SUFFIX} \
                               -x src/518-hardlinkstresstest                   \
                                  src/585-xattrs                               \
                                  src/600-securecvmfs                          \
                                  src/602-libcvmfs                             \
                                  src/628-pythonwrappedcvmfsserver             \
                                  --                                           \
                                  src/5*                                       \
                                  src/6*                                       \
                                  src/7*                                       \
                                  src/8*                                       \
                               || retval=1
```
Tests are blacklisted by listing them after `-x` and before `--` in the invocation of `run.sh`

The test should be blacklisted on the following platforms:
```
fedora_test.sh
slc5_i386_test.sh
slc5_x86_64_test.sh
ubuntu_s3_test.sh
ubuntu_test.sh
```

Thanks!